src/NUnitEngine/nunit.engine.api/nunit.engine.api.csproj

```diff
@@ -60,7 +60,7 @@
       <Link>TestSelectionParserException.cs</Link>
     </Compile>
     <Compile Include="..\EngineApiVersion.cs">
-      <Link>EngineApiVersion.cs</Link>
+      <Link>Properties\EngineApiVersion.cs</Link>
```

Comment:
I've been moving the various assembly info files into the Properties so that they are grouped together and where people would expect them. I took the opportunity here to finish doing it in all the projects.